install_config/configuring_authentication.adoc

```diff
@@ -977,8 +977,12 @@ request header value. This is similar to how
 link:https://access.redhat.com/documentation/en-US/OpenShift_Enterprise/2/html/Deployment_Guide/Configuring_OpenShift_Enterprise_Authentication.html[the remote user plug-in in OpenShift Enterprise 2] allowed administrators to
 provide Kerberos, LDAP, and many other forms of enterprise authentication.
 
+[NOTE]
+====
 You can also use the request header identity provider for advanced configurations
-such as link:https://github.com/openshift/request-header-saml-service-provider[SAML authentication].
+such as the community supported link:https://github.com/openshift/request-header-saml-service-provider[SAML authentication].
+Note that SAML authentication is not supported in Red Hat production service.
```

Comment:
"in Red Hat production service" sounds a little weird to me. Is it supported by Red Hat sometimes, just not in production? If that's the case I'd say "is not supported by Red Hat in production". But if it's not supported by Red Hat at all (since it's community-supported), then I'd just say "is not supported by Red Hat."

Reply:
:+1: